pyproject.toml

```diff
@@ -3,11 +3,40 @@ name = "lagrangebench"
 version = "0.0.2"
 description = "LagrangeBench: A Lagrangian Fluid Mechanics Benchmarking Suite"
 authors = [
+    "Artur Toshev, Gianluca Galletti <artur.toshev@tum.de>"
+]
```

Comment:
Let's see if this works out on pypi

Reply:
These guys do the same: https://github.com/atong01/conditional-flow-matching/blob/main/setup.py
That's the result https://pypi.org/project/torchcfm/